src/tests/Interop/NativeLibrary/API/NativeLibraryTests.csproj

```diff
@@ -26,8 +26,18 @@
     <ItemGroup>
       <AssembliesToCopy Include="$(OutDir)/libNativeLibrary.*" />
       <AssembliesToCopy Include="$(OutDir)/NativeLibrary.*" />
-      <AssembliesToCopy Include="$(OutDir)/$(TargetName).dll" />
     </ItemGroup>
     <Copy SourceFiles="@(AssembliesToCopy)" DestinationFiles="@(AssembliesToCopy -> '$(NativeLibrarySubdirectory)/%(Filename)$(FileNameSuffix)%(Extension)')" />
   </Target>
+
+  <Target Name="SetUpSubdirectoryManaged" AfterTargets="Build" Condition="'$(__SkipManaged)' != '1'">
```

Comment:
Is the `__SkipManaged` condition actually necessary? Doesn't this keying off of the `Build` target handle that?

Reply:
Removed condition statement